### PR TITLE
Upgrade to 5.0.4

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 5.0.4
- Cleaning up

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/snipeit_ynh%20PR15%20(ericgaspar)/)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -28,33 +28,37 @@
...
             },
             {
                 "name": "language",
                 "type": "string",
                 "ask": {
-                    "en": "Choose the application language"
+                    "en": "Choose the application language",
+                    "fr": "Choisissez la langue de l'application"
                 },
-                "choices": ["en", "en-GB", "af", "ar", "bg", "zh-CN", "zh-TW", "hr", "cs", "da", "nl", "et", "fi", "fr", "de", "el", "he", "hu", "id", "en-ID", "ga-IE", "it", "ja", "ko", "lv", "lt", "ms", "mi", "mn", "no", "fa", "pl", "pt-PT", "pt-BR", "ro", "ru", "es-ES", "es-CO", "sv-SE", "ta", "tr", "vi", "zu"],
```
Comment (anmol26s) [inline manifest.json:57]: Do we prefer only English and French?

**Review by anmol26s — COMMENTED** [review]
Looks good other then the language part.

File: manifest.json (hunk 1 above)
Comment (ericgaspar) [reply manifest.json:57]: I just thought maybe it was a long list. We can set 6 or 7 languages or if you want  I can put the list back as it was ...

Comment (anmol26s) [reply manifest.json:57]: But it feels good if user find out his/her language is supported while installation. But in larger picture it make sense if we don't have the language in the app package then why have it as option. And user can change it anytime from web interface.
We can go with en or fr only. Many apps can implement this, but maintaining it is difficult as new languages are added every now and then. There maybe new addition in Snipeit language pack which is not here :D.

**Review by anmol26s — APPROVED** [review]

**Pull request merged**